src/main/drivers/pwm_output_dshot_hal_hal.c

```diff
@@ -369,8 +378,8 @@ P    -    High -     High -
         memset(motor->timer->dmaBurstBuffer, 0, DSHOT_DMA_BUFFER_SIZE * 4 * sizeof(uint32_t));
 
         /* Set hdma_tim instance */
-        motor->timer->hdma_tim.Instance = timerHardware->dmaTimUPRef;
-        motor->timer->hdma_tim.Init.Request = timerHardware->dmaTimUPChannel;
+        motor->timer->hdma_tim.Instance = dmaRef;
```

Comment:
not strictly related to this pr, but why `dmaRef` (aka DMA reference) and not `dmaInstance` ?  I'm not sure about the `reference` suffix at all.  it feels a bit like adding `ptr` to the end of variable names which we don't do.

Reply:
Yeah, ever since I've started to work on CF/BF/iNav, I've always wondered when and why `ref` was used to indicate a pointer to a hardware module instance, but kept using it believing there's a good reason why...

Reply:
:thinking:

Reply:
what should we do then? 
a) introduce more `ref` to keep consistency?
b) change `dmaTimUPRef` to `dmaTimUpdateInstance` now and leave other `ref` for another day.
c) change `dmaTimUPRef` to `dmaTimUpdateInstance` now and fix the other `ref` in another PR now?

Reply:
I'll leave this to another cleanup PR.